api-flask/app/database/alert_database.py

```diff
@@ -1,63 +1,66 @@
-"""Alert database access functions."""
 import logging
 from os import getenv
 
 from app.database.alert_model import AlertModel
 
+from psycopg2 import OperationalError
+
 from sqlalchemy import create_engine
 from sqlalchemy.orm import sessionmaker
 
 logging.basicConfig(level=logging.INFO)
 logger = logging.getLogger(__name__)
 
-DB_URI = getenv('DATABASE_URL') or 'postgresql://{user}:{password}@{host}:{port}/{database}'.format(
-    host='localhost',
-    port=54321,
-    database=getenv('POSTGRES_DB', 'postgres'),
-    user=getenv('POSTGRES_USER', 'postgres'),
-    password=getenv('POSTGRES_PASSWORD', '/!ChangeMe!')
-)
+DB_URI = 'postgresql://{user}:{password}@{host}:{port}/{database}' \
```

Comment:
Let's keep the `getenv('DATABASE_URL')` and also add POSTGRES_PORT as an env as I think that could be useful in the future

Reply:
And have the defaults like this:

`user=getenv('POSTGRES_USER', 'postgres')`

Reply:
Let me update